intezer_sdk/endpoint_analysis.py

```diff
@@ -62,3 +89,136 @@ def _init_sub_analyses(self):
                                               api=self._api,
                                               verdict=sub_analysis['verdict'])
             self._sub_analyses.append(sub_analysis_object)
+
+    def _send_analyze_to_api(self, **additional_parameters) -> str:
+        try:
```

Comment:
We need to raise here since the current on-premise doesn't support the new scanner and the offline mode